secureli/actions/action.py

```diff
@@ -159,7 +173,7 @@ def _install_secureli(
 
         # pre-install
         new_install = len(detected_languages) == len(install_languages)
-
+        self.action_deps.echo.print(f"BINGOOOOOOO, {new_install}")
```

Comment:
Should this be removed/updated?

Reply:
What do you mean? This is really high quality logging 😉